### PR TITLE
Local bananojs hw u2f

### PR DESCRIPTION
**Pull request by dev-ptera**

### CODE REVIEW

**Comment by dev-ptera** [comment]
I opened up a PR against a "dev" branch instead of master so that after merging it I can run prettier, lint & do some local testing with my ledger device before it goes to master.

File: src/app/app.module.ts [hunk 1]
```diff
@@ -186,6 +189,7 @@ import { MatSliderModule } from '@angular/material/slider';
         ResponsiveMenuModule,
         ScrollingModule,
         SpacerModule,
+        TextFieldModule,
```
Comment (dev-ptera) [inline src/app/app.module.ts:192]: Is this module actually required?  If you remove it, does the app still build and the new signing page work?

Comment (Airtune) [reply src/app/app.module.ts:192]: I had to add this for the multiline textarea to work as expected

Comment (Airtune) [reply src/app/app.module.ts:192]: I made a change to the message being signed so it displays as a multiline textfield.
It just makes the message a bit more readable when it has line breaks.

File: src/app/components/account-settings/account-settings.component.ts [hunk 2]
```diff
@@ -36,6 +36,10 @@ import { AppStateService } from '@app/services/app-state.service';
                     <mat-icon>edit_note</mat-icon>
                     <span>Signing</span>
                 </button>
+                <button mat-menu-item (click)="navigateToSignMessage()">
```
Comment (dev-ptera) [inline src/app/components/account-settings/account-settings.component.ts:39]: I have some questions around why this logic broken out into it's own page when there's already a page dedicated for signing.  Seems kind of weird to split this out into a separate route & a different menu navigation item.

File: package.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
         "@angular/platform-browser": "^15.1.0",
         "@angular/platform-browser-dynamic": "^15.1.0",
         "@angular/router": "^15.1.0",
-        "@bananocoin/bananojs-hw": "^1.5.4",
+        "@bananocoin/bananojs-hw": "../bananojs-hw",
```
Comment (Airtune) [inline package.json:37]: This should be changed back to the online repository once this pull-request is merged and published:
https://github.com/BananoCoin/bananojs-hw/pull/2

File: src/app/pages/signmessage/signmessage.component.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+                            *ngIf="submitRequested"
+                            disabled
+                            mat-flat-button
+                            color="primary"
+                            matTooltip="Sign the message with the account's private key"
+                        >
+                            Sign
+                        </button>
+
+                        <mat-card *ngIf="successfulSubmit == false || successfulSubmit == true" appearance="outlined" style="margin-top: 24px;">
```
Comment (Airtune) [inline src/app/pages/signmessage/signmessage.component.html:84]: Styling for this card is OK but the UI could be improved.

File: src/app/components/account-settings/account-settings.component.ts (hunk 2 above)
Comment (Airtune) [reply src/app/components/account-settings/account-settings.component.ts]: Menu item removed

**Pull request merged**